dist/s_all

```diff
@@ -180,6 +180,10 @@ fi
 status ""  # Clear any remaining status.
 wait       # Call "wait" in any case even if the jobs are done to clear their state.
 
+# Execute Ruff checks at the very end once all the Python files have been generated/processed by the
```

Comment:
I think, this can go before parallel scripts since all generation happens serially before that.
And singe it doesn't change files, it can also be parallel.

Reply:
As discussed, it may change files. Added just before we start executing the parallel scripts: 953d951e4d6331d43e713e78370e961efc34c254.